frontend/src/utils/store.ts

```diff
@@ -29,18 +38,19 @@ export const setupStores = async () => {
   }
 
   mainStore.deployedContracts = await db.deployedContracts.toArray()
-  if ( mainStore.accounts.length < 1) {
+  if (mainStore.accounts.length < 1) {
     await mainStore.generateNewAccount()
   } else {
-    mainStore.accounts = localStorage.getItem('mainStore.accounts') ?  (localStorage.getItem('mainStore.accounts') || '').split(',') : []
+    mainStore.accounts = localStorage.getItem('mainStore.accounts')
+      ? (localStorage.getItem('mainStore.accounts') || '').split(',')
```

Comment:
@bradleySuira should we access local storage from here? Aren't we using the state management for the accounts as well?

Reply:
In this case we feed the accounts state form local storage, that's only to not create a lot of new accounts every time
- First load we don't have any account so we call to the PRC to create a new on
- Then we load the existing accounts from local storage and the user can use them or add new accounts

the only issue is that they are not synced with the Database if the user drop the storage or reset the containers but for now I think is ok not sure in the future because accounts should be created in the client side based on seed phrase maybe if we will use EOA

Reply:
sure in the other pr we have an `accountsStore` that has the list of accounts and the current selected account